File: PYTHON/Ejercicio5.py
'''
Escribe un programa que pida al usuario un número entero y determine si es múltiplo de
3 y/o de 5. Imprime "Es múltiplo de 3", "Es múltiplo de 5" o "No es múltiplo de 3 ni de
5" según corresponda.
'''
n=int(input("Introduce un numero entero: "))
if n%3==0 or n%5==0:
    if n%3==0:
        print("Es multiplo de 3")
    if n%5==0:
        print("Es multiplo de 5")
else:
    print ("No es multiplo de 3 ni de 5")

File: JAVA/Ejercicio5.java
import java.util.Scanner;
public class Ejercicio5 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Introduce un numero entero: ");
        int n = sc.nextInt();
        if (n % 3 == 0 || n % 5 == 0) {
            if (n % 3 == 0) {
                System.out.println("Es múltiplo de 3");
            }
            if (n % 5 == 0) {
                System.out.println("Es múltiplo de 5");
            }
        } else {
            System.out.println("No es múltiplo de 3 ni de 5");
        }
    }
}
